streams/mn.m3u

```diff
@@ -1,5 +1,11 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="MongolynMedee.mn" status="online",Монголын Мэдээ (576p)
-http://103.14.38.107:1935/live/mn2.stream/chunklist.m3u8
+http://103.14.38.107:1935/live/mn2.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="MUONT1.mn" status="online",МҮОНТ-1 (576p)
-http://103.14.38.107:1935/live/mnb.stream/chunklist.m3u8
+http://103.14.38.107:1935/live/mnb.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TV5.mn" status="online",TV5
+http://d1qwew1jz60jqg.cloudfront.net/MYASIATV/TV5/playlist.m3u8
```

Comment:
<img width="716" alt="image" src="https://user-images.githubusercontent.com/7253922/172071916-23d504d8-ad68-4335-83a1-31a807a7db74.png">

Reply:
works for me via browser extension
![snapshot](https://user-images.githubusercontent.com/30985701/172074508-89181dbf-a334-42f7-860e-db2f773acb20.jpg)